packages/rest-typings/src/v1/rooms.ts

```diff
@@ -572,4 +606,10 @@ export type RoomsEndpoints = {
 			discussions: IRoom[];
 		}>;
 	};
+
+	'/v1/rooms.isUserMuted': {
+		GET: (params: { userId: string; roomId: string } | { username: string; roomId: string }) => {
```

Comment:
```suggestion
		GET: (params: RoomsIsUserMuted) => {
```
You can use the type defined above.